apps/events/viewsets/events.py

```diff
@@ -31,7 +31,7 @@ class EventViewSet(viewsets.ModelViewSet):
     permission_classes = [IsAuthenticatedOrReadOnly, EventPermissions]
     filter_backends = [TrigramSimilaritySearchFilter, OrderingFilter, DjangoFilterBackend]
     filterset_class = EventFilter
-    search_fields = ['name', 'description', 'address', 'tags__name', 'category__name', 'city']
+    search_fields = ['name', 'description', 'address', 'tags__name', 'category__name',]
```

Comment:
we need place_id instead of city here so we can filter events

Reply:
and maybe location from city_location

Reply:
added search fields